src/com/facebook/buck/go/GoDescriptors.java

```diff
@@ -48,14 +72,13 @@ static BuildTarget createTransitiveSymlinkTreeTarget(BuildTarget source) {
         .build();
   }
 
-  static GoSymlinkTree requireDirectSymlinkTreeRule(BuildRuleParams sourceParams,
```

Comment:
We use the "require" method naming convention for methods which only create the build if it doesn't already exist.  Since this one (and others below) just unconditionally always creates a new build rules, use "create" instead.